packages/block-editor/src/components/block-inspector/index.js

```diff
@@ -127,19 +129,20 @@ const BlockInspectorSingleBlock = ( {
 				</div>
 			) }
 			<InspectorControls.Slot bubblesVirtually={ bubblesVirtually } />
+			<InspectorControls.Slot
+				group="dimensions"
```

Comment:
@mtias mentioned that there are still some design explorations pending that might impact the API a bit (adding tabs and groups). So we might want to start with `__experimentalGroup` to be able to change it later if needed.

Reply:
✅ Done. Thanks.